dev/com.ibm.ws.jpa_22_fat/publish/servers/JPADefaultDataSourceServer_JTA/bootstrap.properties

```diff
@@ -0,0 +1,11 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
```

Comment:
probably want some sort of JPA trace enabled here in order to debug any test failures that may occur. Based on the tests, I think `*=info:com.ibm.ws.jpa.management=all:com.ibm.ws.jpa.container.osgi.internal=all` would be good. 
Same comment also applies to the other 2 servers added in this PR

Reply:
Done